### PR TITLE
Simplified Chinese Translation of Rocky Documentation

### PR DESCRIPTION
**Pull request by yangxuan74**
add file shh公私钥.md and modify wrongly written

### CODE REVIEW

**Review by tjyang — COMMENTED** [review]
this change is a English typo correction.

**Comment by yangxuan74** [comment]
> This file should be ssh_公私钥.md instead. We dealt with the english typo on another pull request. I'd like to get another reader for this before merging into main.

I don't think it's necessary to add "_" between the characters in the file name under Chinese environment,because writing like this will not cause ambiguity.However, ssh should be capitalized as it is an abbreviation.

**Comment by calderds** [comment]
Sounds good. I just mentioned the ssh part since it's "shh" in the pull request. @tjyang read the document over, so we should be good to go once we fix the naming convention.

I think once we get a larger translation cohort, we'll go back and look at the older work we've already put out, so we can merge with main knowing it's a first draft.

**Comment by yangxuan74** [comment]
Strongly agree, good works need to be revised many times.

File: zh_cn/rocky/8/guides/apache-sites-enabled.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+
+此过程首先生成私钥和 CSR（表示证书签名请求），然后将 CSR 提交给证书颁发机构以购买 SSL 证书。生成这些密钥的过程有些复杂，因此它有自己的文档。
+
+如果您不熟悉生成 SSL 密钥，请查看：[生成 SSL 密钥](ssl_keys_https.md)
+
+### 密钥和证书的位置
+
+现在您已经拥有了密钥和证书文件，此时需要将它们按逻辑放置在 Web 服务器上的文件系统中。正如在上面示例配置文件中所看到的，将 Web 文件放置在 _/var/www/sub-domains/com.ourownwiki.www/html_ 中。
+
+希望将证书和密钥文件放在域（domain）中，而不是放在文档根（document root）目录中（在本例中是 _html_ 文件夹）。
```
Comment (bobby285271) [inline zh_cn/rocky/8/guides/apache-sites-enabled.md:103]:
```suggestion
我们建议您将证书和密钥文件放在域（domain）中，而不是放在文档根（document root）目录中（在本例中是 _html_ 文件夹）。
```

File: zh_cn/rocky/8/guides/apache-sites-enabled.md [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+
+绝不希望证书和密钥有可能暴露在网络上。那会很糟糕！
```
Comment (bobby285271) [inline zh_cn/rocky/8/guides/apache-sites-enabled.md:105]:
```suggestion
如果不这样做，证书和密钥有可能暴露在网络上。那会很糟糕！
```

File: zh_cn/rocky/8/guides/apache-sites-enabled.md [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+
+相反，将在文档根目录之外为 SSL 文件创建一个新目录结构：
```
Comment (bobby285271) [inline zh_cn/rocky/8/guides/apache-sites-enabled.md:107]:
```suggestion
我们建议的做法是，将在文档根目录之外为 SSL 文件创建新目录：
```

File: zh_cn/rocky/8/guides/apache-sites-enabled.md [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+
+```
+<VirtualHost *:80>
+        ServerName www.ourownwiki.com 
+        ServerAdmin username@rockylinux.org
+        Redirect / https://www.ourownwiki.com/
+</VirtualHost>
+```
+
+这意味着要将任何常规 Web 请求发送到 https 配置。上面显示的 apache “Redirect”选项可以在所有测试完成后更改为“Redirect permanent”，您可以看到站点按照您希望的方式运行。此处选择的“Redirect”是临时重定向。
```
Comment (bobby285271) [inline zh_cn/rocky/8/guides/apache-sites-enabled.md:143]:
```suggestion
这意味着要将任何常规 Web 请求发送到 https 配置。上面显示的 apache “Redirect”选项可以在所有测试完成后更改为“Redirect permanent”，此时站点应该就会按照您希望的方式运行。此处选择的“Redirect”是临时重定向。
```

File: zh_cn/rocky/8/guides/cron_jobs_howto.md [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+## 创建自定义 cron
+
+当然，无论出于何种原因，自动化时间都不能很好地为您工作，那么您可以创建自定义时间。本示例中，假设您以 root 用户身份执行此操作。为此，[请参见假定](##-assumptions)，键入以下内容：
+
+`crontab -e`
+
+这将调出 root 用户的 crontab，就像它此时存在于您选择的编辑器中一样，可能如下所示。请继续阅读以下注释，因为它包含接下来将使用的每个字段的描述：
+
+```
+# 编辑此文件以引入要由cron运行的任务。
```
Comment (bobby285271) [inline zh_cn/rocky/8/guides/cron_jobs_howto.md:42]:
```suggestion
# 编辑此文件以引入要由 cron 运行的任务。

```

File: zh_cn/rocky/8/guides/cron_jobs_howto.md [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+# 
+# 一行定义一个要运行的任务
+# 在一行中用不同的字段指示何时运行该任务
+# 以及为该任务运行的命令
+# 
+# 要定义时间，可以为
+# 分钟（m）、小时（h）、月日（dom）、月（mon）和星期（dow）
+# 提供具体值，或者在这些字段中使用“*”（表示“任何”）。
+# 
+# 注意，将根据cron的系统守护程序的时间和时区概念启动任务。
```
Comment (bobby285271) [inline zh_cn/rocky/8/guides/cron_jobs_howto.md:52]:
```suggestion
# 注意，将根据 cron 的系统守护程序的时间和时区概念启动任务。

```

File: zh_cn/rocky/8/guides/cron_jobs_howto.md [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+# 
+# 作业的输出（包括错误）通过电子邮件发送给
+# crontab文件所属的用户（除非重定向）。
```
Comment (bobby285271) [inline zh_cn/rocky/8/guides/cron_jobs_howto.md:55]:
```suggestion
# crontab 文件所属的用户（除非重定向）。

```

File: zh_cn/rocky/8/guides/cron_jobs_howto.md [hunk 8]
```diff
@@ -0,0 +1,126 @@
...
+# cron
+# 例如，每周一上午5点运行所有用户帐户的备份：
```
Comment (bobby285271) [inline zh_cn/rocky/8/guides/cron_jobs_howto.md:57]:
```suggestion
# 例如，每周一上午 5 点运行所有用户帐户的备份：

```

File: zh_cn/rocky/8/guides/cron_jobs_howto.md [hunk 9]
```diff
@@ -0,0 +1,126 @@
...
+# 0 5 * * 1 tar -zcf /var/backups/home.tgz /home/
+# 
+# 有关更多信息，请参见 crontab（5）和 cron（8）的手册页
+# 
+# m h  dom mon dow   command
+```
+
+注意，这个特定的 crontab 文件内置了一些自己的文档。并非总是如此。在容器或极简操作系统上修改 crontab 时，crontab 将是一个空文件，除非已经在其中放置了条目。
+
+假设有一个备份脚本，希望在晚上 10 点运行。crontab 使用 24 小时制，因此应该是 22:00。假设备份脚本名为“backup”，并且当前位于 _/usr/local/sbin_ 目录中。
```
Comment (bobby285271) [inline zh_cn/rocky/8/guides/cron_jobs_howto.md:67]:
```suggestion
假设有一个备份脚本，你希望系统在晚上 10 点运行它。crontab 使用 24 小时制，因此应该是 22:00。假设备份脚本名为“backup”，并且当前位于 _/usr/local/sbin_ 目录中。

```

File: zh_cn/rocky/8/guides/cron_jobs_howto.md [hunk 10]
```diff
@@ -0,0 +1,126 @@
...
+
+Note: Remember that this script needs to also be executable (`chmod +x`) in order for the cron to run it. To add the job, we would:注意：这个脚本也需要是可执行的（`chmod+x`），cron 才能运行它。要添加作业：
```
Comment (bobby285271) [inline zh_cn/rocky/8/guides/cron_jobs_howto.md:69]:
```suggestion
注意：这个脚本也需要是可执行的（`chmod+x`），cron 才能运行它。要添加作业：

```

File: zh_cn/rocky/8/guides/cron_jobs_howto.md [hunk 11]
```diff
@@ -0,0 +1,126 @@
...
+
+`crontab -e`
+
+ “crontab”代表“cron table”，文件的格式实际上是松散的表布局。现在在 crontab 中，转到文件的底部并添加新条目。如果您使用 vi 作为默认的系统编辑器，那么可以通过以下键来完成：
+
+`Shift : $`
+
+现在，您位于文件的底部，插入一行并键入简短的注释以描述条目的内容。注释是通过在行首添加“＃”来完成的：
+
+`# 每晚10点备份系统`
```
Comment (bobby285271) [inline zh_cn/rocky/8/guides/cron_jobs_howto.md:79]:
```suggestion
`# 每晚 10 点备份系统`

```

File: zh_cn/rocky/8/guides/cron_jobs_howto.md [hunk 12]
```diff
@@ -0,0 +1,126 @@
...
+
+现在按回车键。您应该仍然处于插入模式，因此下一步是添加条目。如以上空注释的 crontab 所示，**m** 表示分钟，**h** 表示小时，**dom** 表示月日，**mon** 表示月，**dow** 表示星期。
+
+要在每晚 10:00 运行备份脚本，条目如下所示：
+
+`00  22  *  *  *   /usr/local/sbin/backup`
+
+这表示在每月、每周和每日的晚上 10 点运行脚本。显然，这是一个非常简单的示例，当您需要详细说明时，情况可能会变得非常复杂。
+
+### crontab的@选项
```
Comment (bobby285271) [inline zh_cn/rocky/8/guides/cron_jobs_howto.md:89]:
```suggestion
### crontab 的 @ 选项

```

File: zh_cn/rocky/8/guides/cron_jobs_howto.md [hunk 13]
```diff
@@ -0,0 +1,126 @@
...
+
+如本文档上面的[轻松入门](##-starting-easy)部分所述，cron 点目录中的脚本在特定的时间运行。@选项提供使用更自然的定时能力。@选项包括：
```
Comment (bobby285271) [inline zh_cn/rocky/8/guides/cron_jobs_howto.md:91]:
```suggestion
如本文档上面的[轻松入门](##-starting-easy)部分所述，cron 点目录中的脚本在特定的时间运行。@ 选项提供使用更自然的定时能力。@ 选项包括：
```

File: zh_cn/rocky/8/guides/mirroring_lsyncd.md [hunk 14]
```diff
@@ -0,0 +1,232 @@
+# Rocky Linux - 镜像解决方案 - lsycnd
+
+## 准备工作
+
+以下是您需要了解并遵循本指南的所有内容：
```
Comment (bobby285271) [inline zh_cn/rocky/8/guides/mirroring_lsyncd.md:5]:
```suggestion
在继续阅读这篇指南之前，您需要准备或了解以下的内容：
```

File: zh_cn/rocky/8/guides/mirroring_lsyncd.md [hunk 15]
```diff
@@ -0,0 +1,232 @@
...
+## 准备工作
+
+以下是您需要了解并遵循本指南的所有内容：
+
+* 运行 Rocky Linux 的计算机。
+* 熟练从命令行修改配置文件。
+* 了解如何使用命令行编辑器（本文使用 vi，但您也可以使用其他编辑器）。
+* 您需要 root 用户访问权限，并且最好以 root 用户身份登录终端。
+* SSH 公私密钥对。
+* Fedora的 EPEL 仓库
```
Comment (bobby285271) [inline zh_cn/rocky/8/guides/mirroring_lsyncd.md:12]:
```suggestion
* Fedora 的 EPEL 仓库

```

File: zh_cn/rocky/8/guides/mirroring_lsyncd.md [hunk 16]
```diff
@@ -0,0 +1,232 @@
...
+
+即便如此，它还是值得任何系统管理员认真学习。
+
+*lsyncd* 的最佳描述来自其 man 手册页。稍微解释一下，lsyncd 是一种轻量级实时镜像解决方案，相对易于安装。它不需要新的文件系统或块设备，也不会影响本地文件系统的性能。简而言之，它镜像文件。
+
+lsyncd 监视本地目录树事件监视器接口（inotify）。它在几秒钟内聚合并组合事件，然后生成一个（或多个）进程来同步更改。默认情况下，进程是 rsync。
+
+在本文中，带有原始文件的系统称为“master（主）”，而与之同步的系统将称为“target（目标）”。实际上，，使用 lsyncd 通过非常仔细地指定要同步的目录和文件是有可能完全镜像服务器。真是太好了！
+
+对于远程同步，您还需要设置[Rocky Linux SSH 公私钥对](RL_ssh_public_private_keys.md)。此处的示例使用 SSH（端口22）。
```
Comment (bobby285271) [inline zh_cn/rocky/8/guides/mirroring_lsyncd.md:28]:
```suggestion
对于远程同步，您还需要设置 [Rocky Linux SSH 公私钥对](RL_ssh_public_private_keys.md)。此处的示例使用 SSH（端口 22）。

```

File: zh_cn/rocky/8/guides/mirroring_lsyncd.md [hunk 17]
```diff
@@ -0,0 +1,232 @@
...
+
+安装 RPM 版本相对容易。您唯一需要首先安装的是 Fedora 的 EPEL 软件库。只需一条命令即可完成此操作：
+
+`dnf install https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm`
+
+然后，只需安装 lsyncd，任何缺少的依赖项都将与它一起安装：
+
+`dnf install lsyncd`
+
+That's it!就这样！
```
Comment (bobby285271) [inline zh_cn/rocky/8/guides/mirroring_lsyncd.md:44]:
```suggestion
就这样！
```

File: zh_cn/rocky/8/guides/mirroring_lsyncd.md [hunk 18]
```diff
@@ -0,0 +1,232 @@
...
+
+## Installing lsycnd - Source Method安装 lsycnd —— 源码方法
```
Comment (bobby285271) [inline zh_cn/rocky/8/guides/mirroring_lsyncd.md:46]:
```suggestion
## 安装 lsycnd —— 源码方法

```

File: zh_cn/rocky/8/guides/mirroring_lsyncd.md [hunk 19]
```diff
@@ -0,0 +1,232 @@
...
+
+Installing from source is not as bad is it sounds. Just follow this guide and you will be up and running in no time!从源码安装并不复杂，只需遵循本指南，您将立即启动并运行！
```
Comment (bobby285271) [inline zh_cn/rocky/8/guides/mirroring_lsyncd.md:48]:
```suggestion
从源码安装并不复杂，只需遵循本指南，相信您能很快完成一切事情！
```

File: zh_cn/rocky/8/guides/mirroring_lsyncd.md [hunk 20]
```diff
@@ -0,0 +1,232 @@
...
+
+### Install Dependencies安装依赖
```
Comment (bobby285271) [inline zh_cn/rocky/8/guides/mirroring_lsyncd.md:50]:
```suggestion
### 安装依赖

```

File: zh_cn/rocky/8/guides/mirroring_lsyncd.md [hunk 21]
```diff
@@ -0,0 +1,232 @@
...
+
+We will need some dependencies: a few that are required by lsyncd itself, and a few that are required to build packages from source. Use this command on your Rocky Linux machine to make sure you have the dependencies you need. If you are going to be building from source, it's a good idea to have all of the development tools installed:将需要一些依赖项：lsyncd 本身需要的一些依赖项，以及从源码构建软件包所需要的一些依赖项。在 Rocky Linux 机器上使用此命令，以确保您具有所需的依赖项。如果要从源码进行构建，则最好安装所有开发工具：
```
Comment (bobby285271) [inline zh_cn/rocky/8/guides/mirroring_lsyncd.md:52]:
```suggestion
我们需要安装下面的一些依赖项：lsyncd 本身需要的依赖项，以及从源码构建软件包所需要的依赖项。在 Rocky Linux 机器上使用此命令，以确保您具有所需的依赖项。如果要从源码进行构建，则最好安装所有开发工具：
```

File: zh_cn/rocky/8/guides/mirroring_lsyncd.md [hunk 22]
```diff
@@ -0,0 +1,232 @@
...
+
+`dnf install groupinstall 'Development Tools'`
+
+以下是 lsyncd 本身及其构建过程所需的依赖项：
+
+`dnf install lua lua-libs lua-devel cmake unzip wget rsync`
+
+### 下载 lsycnd 并构建
+
+接下来，需要源码：
```
Comment (bobby285271) [inline zh_cn/rocky/8/guides/mirroring_lsyncd.md:62]:
```suggestion
接下来，获取源码：

```

File: zh_cn/rocky/8/guides/mirroring_lsyncd.md [hunk 23]
```diff
@@ -0,0 +1,232 @@
...
+
+这将创建一个名为“lsyncd-master”的目录。切换到该目录并创建一个名为 build 的目录：
+
+`cd lsyncd-master`
+
+And then:
+
+`mkdir build`
+
+现在再次更改目录以位于构建目录中：
```
Comment (bobby285271) [inline zh_cn/rocky/8/guides/mirroring_lsyncd.md:78]:
```suggestion
现在再次更改目录以进入构建目录：
```

File: zh_cn/rocky/8/guides/mirroring_lsyncd.md [hunk 24]
```diff
@@ -0,0 +1,232 @@
...
+cmake ..
+make
+make install
+```
+
+完成后，您应该已经安装了 lsyncd 二进制文件并可以在 */usr/local/bin* 中使用。
+
+# lsycnd Systemd 服务
+
+两种安装方法都不会创建用于重启时启动 lsyncd 的 systemd 服务。我们希望能够做到这一点，因为如果您正在镜像文件，则不希望因忘记手动启动服务而使镜像处于脱机状态，。
```
Comment (bobby285271) [inline zh_cn/rocky/8/guides/mirroring_lsyncd.md:94]:
```suggestion
两种安装方法都不会创建用于重启时启动 lsyncd 的 systemd 服务。我们希望能够做到这一点，因为如果您正在镜像文件，肯定不希望因忘记手动启动服务而使镜像处于脱机状态。

```

File: zh_cn/rocky/8/guides/mirroring_lsyncd.md [hunk 25]
```diff
@@ -0,0 +1,232 @@
...
+
+如果要添加多个目录进行同步，则需要为每个目录重复整个"sync"部分，包括所有开始括号和结束括号。
+
+## lsyncd.exclude 文件
+
+如前所述，"excludeFrom"文件必须存在，所以现在创建它：
+
+`touch /etc/lsyncd.exclude`
+
+如果要在计算机上同步 /etc 文件夹，那么应该忽略许多文件和/或目录。每个排除的文件或目录都简单地列在文件中，每行一个，如下所示：
```
Comment (bobby285271) [inline zh_cn/rocky/8/guides/mirroring_lsyncd.md:186]:
```suggestion
如果要在计算机上同步 /etc 文件夹，那么应该会希望忽略许多文件和目录。你只需要将你希望排除的文件和目录简单地列在文件中，每行一个，如下所示：

```

File: zh_cn/rocky/8/guides/mirroring_lsyncd.md [hunk 26]
```diff
@@ -0,0 +1,232 @@
...
+```
+/etc/hostname
+/etc/hosts
+/etc/networks
+/etc/fstab
+```
+
+## 测试和启动
+
+现在已经完成了其他所有设置，可以对其进行全部测试。首先，请确保 systemd lsyncd.service 将启动：
```
Comment (bobby285271) [inline zh_cn/rocky/8/guides/mirroring_lsyncd.md:197]:
```suggestion
现在已经完成了其他所有设置，可以对其进行全部测试。首先，请确保 systemd lsyncd.service 可以启动：

```

File: zh_cn/rocky/8/guides/mirroring_lsyncd.md [hunk 27]
```diff
@@ -0,0 +1,232 @@
...
+
+`systemctl start lsyncd`
+
+如果执行此命令后没有错误出现，请检查服务状态，以确保：
```
Comment (bobby285271) [inline zh_cn/rocky/8/guides/mirroring_lsyncd.md:201]:
```suggestion
如果执行此命令后没有错误出现，再进一步检查服务状态，以确保一切无误：

```

File: zh_cn/rocky/8/guides/mirroring_lsyncd.md [hunk 28]
```diff
@@ -0,0 +1,232 @@
...
+
+如果显示服务正在运行，请使用 tail 查看两个日志文件的结尾，并确保一切都正常显示：
+
+`tail /var/log/lsyncd.log`
+
+然后：
+
+`tail /var/log/lsyncd-status.log`
+
+假设一切正确，请导航到 `/home/[user]` 目录，其中 `[user]` 是计算机上的用户，并使用 *touch* 在其中创建新文件。
```
Comment (bobby285271) [inline zh_cn/rocky/8/guides/mirroring_lsyncd.md:213]:
```suggestion
假设一切正确，进入到 `/home/[user]` 目录，其中 `[user]` 是计算机上的用户，并使用 *touch* 在其中创建新文件。

```

File: zh_cn/rocky/8/guides/rsync_ssh.md [hunk 29]
```diff
@@ -0,0 +1,151 @@
+# 使用 rsync 保持两台计算机同步
+
+## 准备工作
+
+以下是您需要了解并遵循本指南的所有准备工作。
```
Comment (bobby285271) [inline zh_cn/rocky/8/guides/rsync_ssh.md:5]:
```suggestion
在继续阅读这篇指南之前，您需要准备或了解以下的内容：
```

File: zh_cn/rocky/8/guides/rsync_ssh.md [hunk 30]
```diff
@@ -0,0 +1,151 @@
...
+
+* -r, 递归目录
+* -l, 将符号链接保持为符号链接
+* -p, 保留权限
+* -t, 保留修改时间
+* -g, 保留 group-
+* -o, 保留所有者
+* -D, 保留设备文件
+
+The only other options that we need to specify in this example is:在此示例中，需要指定的唯一其他选项是：
```
Comment (bobby285271) [inline zh_cn/rocky/8/guides/rsync_ssh.md:47]:
```suggestion
在此示例中，需要指定的唯一其他选项是：
```

File: zh_cn/rocky/8/guides/rsync_ssh.md [hunk 31]
```diff
@@ -0,0 +1,151 @@
...
+# Output of the crontab jobs (including errors) is sent through
+# email to the user the crontab file belongs to (unless redirected).
+# 
+# For example, you can run a backup of all your user accounts
+# at 5 a.m every week with:
+# 0 5 * * 1 tar -zcf /var/backups/home.tgz /home/
+# 
+# For more information see the manual pages of crontab(5) and cron(8)
+# 
+# m h  dom mon dow   command
```
Comment (bobby285271) [inline zh_cn/rocky/8/guides/rsync_ssh.md:137]: 前面 `zh_cn/rocky/8/guides/cron_jobs_howto.md` 是把这段注释也翻译了的，看看这里要不要也翻译（或者前面不翻译也可）。

File: zh_cn/rocky/8/guides/ssh_public_private_keys.md [hunk 32]
```diff
@@ -0,0 +1,92 @@
...
+
+`ssh -l root web.ourourdomain.com` 
+
+假设顺利登录到三台计算机上，那么下一步就是将公钥发送到每个服务器：
+
+`scp .ssh/id_rsa.pub root@web.ourourdomain.com:/root/` 
+
+对每台计算机重复此步骤。
+
+在每个打开的终端窗口中，输入以下命令，将看到 *id_rsa.pub*：
```
Comment (bobby285271) [inline zh_cn/rocky/8/guides/ssh_public_private_keys.md:60]:
```suggestion
在每个打开的终端窗口中，输入以下命令，您应该看到 *id_rsa.pub*：
```

File: zh_cn/rocky/8/guides/ssl_keys_https.md [hunk 33]
```diff
@@ -0,0 +1,123 @@
+# 生成 SSL 密钥
+
+# 准备工作
+
+* 一个工作站和一个运行 RockyLinux 的服务器。
```
Comment (bobby285271) [inline zh_cn/rocky/8/guides/ssl_keys_https.md:5]:
```suggestion
* 一个工作站和一台运行 Rocky Linux 的服务器。
```

File: zh_cn/rocky/8/guides/ssl_keys_https.md [hunk 34]
```diff
@@ -0,0 +1,123 @@
...
+
+# 简介
+
+如今，每个网站都应使用 SSL（安全套接字协议层）证书运行。本文将指导您生成网站私钥，然后从中生成用于购买新证书的 CSR（证书签名请求）。
+
+## 生成私钥
+
+对于新手而言，SSL 私钥可以有不同的大小（以位为单位），其大小决定了破解的难易程度。
+
+截至2021年，推荐的网站私钥大小仍为 2048 位。您可以使用更多位的密钥，但是将密钥大小从 2048 位增加一倍至 4096 位只会使安全性提高约 16%，但需要更多空间以存储密钥，并且在处理密钥时会导致 CPU 负载增高。
```
Comment (bobby285271) [inline zh_cn/rocky/8/guides/ssl_keys_https.md:19]:
```suggestion
截至 2021 年，推荐的网站私钥大小仍为 2048 位。您可以使用更多位的密钥，但是将密钥大小从 2048 位增加一倍至 4096 位只会使安全性提高约 16%，但需要更多空间以存储密钥，并且在处理密钥时会导致 CPU 负载增高。

```

File: zh_cn/rocky/8/guides/ssl_keys_https.md [hunk 35]
```diff
@@ -0,0 +1,123 @@
...
+
+示例使用的域是 ourownwiki.com。注意，您需要提前购买和注册域名。您可以通过多个“注册商”购买域名。
+
+如果您没有运行自己的 DNS（域名系统），您可以使用 DNS 托管提供类似功能。DNS 将您的命名域转换为 Internet 可以理解的数字（IP地址、IPv4 或 IPv6）。这些 IP 地址将是网站的实际托管位置。
+
+使用 openssl 生成密钥：
+
+`openssl genrsa -des3 -out ourownwiki.com.key.pass 2048`
+
+注意，将密钥命名为 .pass 扩展名。这是因为一旦执行此命令，它就会要求您输入密码。输入一个您可以记住的简单密码，因为很快就会移除此密码：
```
Comment (bobby285271) [inline zh_cn/rocky/8/guides/ssl_keys_https.md:38]:
```suggestion
注意，将密钥命名为 .pass 扩展名。这是因为一旦执行此命令，它就会要求您输入密码。输入一个您可以记住的简单密码，因为我们稍后就会移除这个密码：

```

File: zh_cn/rocky/8/guides/ssh_public_private_keys.md [hunk 36]
```diff
@@ -0,0 +1,92 @@
+# Rocky Linux - SSH 公钥和私钥
+
+## 准备工作
+
+* 熟悉命令行操作。
+* Rocky Linux 服务器和/或工作站已安装 *openssh*。
```
Comment (bobby285271) [inline zh_cn/rocky/8/guides/ssh_public_private_keys.md:6]:
```suggestion
* 安装有 *openssh* 的 Rocky Linux 服务器或工作站。
```

**Review by bobby285271 — COMMENTED** [review]
Thanks for helping ! 😉

File: zh_cn/rocky/8/guides/apache-sites-enabled.md [hunk 37]
```diff
@@ -0,0 +1,210 @@
...
+# Apache Web 服务器多站点设置
+
+Rocky Linux 提供了许多方法来设置网络站点。Apache 只是其中的一种方法，用于在单台服务器上进行多站点设置。尽管此方法是为多站点服务器设计的，但它也可以作为单个站点服务器的基本配置。 
+
+历史事实：此服务器设置似乎是从基于 Debian 的系统开始的，但它完全适合于任何运行 Apache 的 Linux 操作系统。
+
+## 准备工作
+
+* 一台运行 Rocky Linux 的服务器
+* 了解命令行和文本编辑器（选择您喜欢的编辑器，本示例使用 *vi*。）
```
[the remark this answers is not in the document]
Comment (yangxuan74) [reply zh_cn/rocky/8/guides/apache-sites-enabled.md:14]: 了解命令行和文本编辑器（本示例使用 *vi*，但您可以选择任意您喜欢的编辑器）

File: zh_cn/rocky/8/guides/apache-sites-enabled.md [hunk 38]
```diff
@@ -0,0 +1,210 @@
...
+    * 如果您想了解 vi 文本编辑器，[此处有一个简单教程](https://www.tutorialspoint.com/unix/unix-vi-editor.html)。
+* 有关安装和运行 Web 服务的基本知识
+
+## 安装 Apache
+
+站点可能需要其他软件包。例如，几乎肯定需要 PHP，也可能需要一个数据库或其他包。从 Rocky Linux 仓库获取 PHP 与 httpd 的最新版本并安装。
+
+谨记，您可能还需要模块，例如 php-bcmath 或 php-mysqlind。Web 应用程序规范应详细说明所需的模块。这些模块可以随时安装。现在将安装 httpd 和 PHP：
```
[the remark this answers is not in the document]
Comment (yangxuan74) [reply zh_cn/rocky/8/guides/apache-sites-enabled.md:22]: 其实中文里不需要那么多人称代词，如您、我们，显得啰嗦，我好多地方都省略了人称代词。并且英文中一会使用you，一会使用we，省略掉一律表示阅读本文的用户就行。以上是我的看法，您觉得呢？

[the remark this answers is not in the document]
Comment (bobby285271) [reply zh_cn/rocky/8/guides/apache-sites-enabled.md:22]: 嗯，我更新了一下这段的建议。

**Review by bobby285271 — APPROVED** [review]
Looks good to me ! 😄

**Comment by calderds** [comment]
What's the status of this? Is it ready to merge, or needs some more time to add more info?

**Comment by EzequielBruni** [comment]
> What's the status of this? Is it ready to merge, or needs some more time to add more info?

That's the thing, I have no idea. If you think it's ready, I'll happily merge it.

**Review by yangxuan74 — COMMENTED** [review]
> > What's the status of this? Is it ready to merge, or needs some more time to add more info?
> 
> That's the thing, I have no idea. If you think it's ready, I'll happily merge it.

I just translate them into Chinese. At present, only one person has reviewed it. There are still many articles that have not been translated, so I don't need to merge them for the time being.

**Comment by calderds** [comment]
I'm going to merge this just so we get the installation guide translated on the site. If there are further updates, we can fix them with a future PR.

**Comment by yangxuan74** [comment]
OK，that can be merged now.&nbsp;Many thanks !&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "rocky-linux/documentation"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年7月14日(星期三) 上午9:33
***@***.***&gt;;
抄送:&nbsp;"@***@***.******@***.***&gt;;
主题:&nbsp;Re: [rocky-linux/documentation] Simplified Chinese Translation of Rocky Documentation (#32)





 
I'm going to merge this just so we get the installation guide translated on the site. If there are further updates, we can fix them with a future PR.

**Pull request merged**